tools/clang/lib/CodeGen/CGExprConstant.cpp

```diff
@@ -918,7 +919,7 @@ class ConstExprEmitter :
         llvm::Constant *destVal = llvm::Constant::getNullValue(destType);
         return destVal;
       }
-    }
+    } break;
```

Comment:
Unreachable or assert here too?

Reply:
The unreachable on line 949 is reached by breaking here.

Reply:
I suppose we could have a more specialized message though